lua/xadmin/commands/util.lua

```diff
@@ -1,14 +1,14 @@
 --- #
 --- # NOCLIP
 --- #
-xAdmin.Core.RegisterCommand("noclip", "Toggle a user's noclip", 30, function(admin, args)
+xAdmin.Core.RegisterCommand("noclip", "Toggle a user's noclip", xAdmin.Config.PowerlevelPermissions["noclip"], function(admin, args)
 	local target = admin
 
 	if args and args[1] then
 		target = xAdmin.Core.GetUser(args[1], admin)
 
 		if not IsValid(target) or admin.isConsole then
-			xAdmin.Core.Msg({Color(46, 170, 200), "[xAdmin] ", color_white, "Please provide a valid target. The following was not recognised: " .. args[1]}, admin)
+			xAdmin.Core.Msg({Color(46, 170, 200), xAdmin.Config.LogPrefix, xAdmin.Config.ColorLogText, "Please provide a valid target. The following was not recognised: " .. args[1]}, admin)
```

Comment:
All the invalid argument prompts for this file have double prexies.  https://i.notgyazo.club/f1B81OBID.png